plugins/modules/rds_instance_info.py

```diff
@@ -359,6 +359,7 @@
     pass  # handled by AnsibleAWSModule
 
 
+@AWSRetry.jittered_backoff()
```

Comment:
I can't quite remember if its enough for use with the paginator to just add this, what I normally do is split out the paginator like this into its own function so that wrapped up calls still get covered:
https://github.com/ansible-collections/community.aws/blob/main/plugins/modules/elb_target_group_info.py#L220